ui/src/tasks/components/TaskRunsPage.tsx

```diff
@@ -1,25 +1,47 @@
 // Libraries
 import React, {PureComponent} from 'react'
-import {withRouter, WithRouterProps} from 'react-router'
+import {connect} from 'react-redux'
 
 // Types
-import {Run} from '@influxdata/influx'
 import {Page} from 'src/pageLayout'
 import TaskRunsList from 'src/tasks/components/TaskRunsList'
+import {AppState} from 'src/types/v2'
 
-// DummyData
-import {taskRuns} from 'src/tasks/dummyData'
+// Actions
+import {getRuns} from 'src/tasks/actions/v2'
+import {Run} from '@influxdata/influx'
+import {RemoteDataState} from 'src/types'
 
-interface Props {
+interface PassedInProps {
   params: {id: string}
+}
+
+interface ConnectedDispatchProps {
+  getRuns: typeof getRuns
+}
+
+interface ConnectedStateProps {
   runs: Run[]
 }
 
-const dummyData = taskRuns
+interface State {
+  loading: RemoteDataState
+}
+
+type Props = PassedInProps & ConnectedDispatchProps & ConnectedStateProps
 
-class TaskRunsPage extends PureComponent<Props & WithRouterProps> {
+class TaskRunsPage extends PureComponent<Props, State> {
+  constructor(props: Props) {
```

Comment:
_nit_ I think a slightly more declarative way of doing this is to use a instance property instead of the `constructor`:

```
class TaskRunsPage extends PureComponent<Props, State> {
  public state: State = {loading: RemoteDataState.NotStarted}

  // ...
```